libs/journeys/ui/src/components/Card/OverlayContent/DownScrollArrow/DownScrollArrow.tsx

```diff
@@ -0,0 +1,49 @@
+import Box from '@mui/material/Box'
+import Fade from '@mui/material/Fade'
+import { keyframes } from '@mui/material/styles'
+import { ReactElement, useEffect, useState } from 'react'
+
+import ChevronDownIcon from '@core/shared/ui/icons/ChevronDown'
+
+interface DownScrollArrowProps {
+  trigger: boolean
+}
+export function DownScrollArrow({
```

Comment:
Would it be a better name to swap scroll and down and also not tie the naming to what icon is used?

```suggestion
interface ScrollDownIndicatorProps {
  trigger: boolean
}
export function ScrollDownIndicator({
```

You could also take the icon in as a prop so that in the future you could use this with another icon instead of that particular down arrow.